app/src/main/res/values/strings.xml

```diff
@@ -1801,7 +1801,7 @@
     <!-- This is the hint text that is shown inline on the hostname field of the create new login page. 'https://www.example.com' intentionally hardcoded here -->
     <string name="add_login_hostname_hint_text">https://www.example.com</string>
     <!-- This is an error message shown below the hostname field of the add login page when a hostname does not contain http or https. -->
-    <string name="add_login_hostname_invalid_text_1">Web address must contain \“https://\“ or \“http://\“</string>
```

Comment:
Does the string ID need to be incremented?

Reply:
Good question. I'm not sure, since this string hasn't been translated yet. I'll check with @Delphine

Reply:
answer: yes, it's better to update the id since it's already been exposed on Pontoon :D

Reply:
We don't need to change the reference in the code too? I can see one here: 
https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/res/layout/fragment_add_login.xml#47

Reply:
Wrote my comment while you pushed. :)

Reply:
It's up there 👆